views/components/Certification.js

```diff
@@ -1,72 +1,396 @@
-        <h4></h4>
-      </header>
-      <div className="grid-container">
-        <ul className="process">
-          <li className="process__step process__step_research">
-            <div className="process__step-icon icon icon_cert is-active"></div>
-          </li>
-          <li className="process__step-info process__step-info_research is-active">
-            <h3>
-              <spana>1</spana>
-              &mdash;Bachelor of Science
-            </h3>
-            <p>
-              Graduating from HCM University of Science, Advanced Program in
-              Computer Science.
-            </p>
-          </li>
-          <li className="process__step process__step_develop">
-            <div className="process__step-icon icon icon_cert"></div>
-          </li>
-          <li className="process__step-info process__step-info_develop">
-            <h3>
-              <spana>2</spana>
-              &mdash;Communicative Certification: TOEFL ibt 85 pts
-            </h3>
-            <p>
-              The TOEFL iBT® test measures your ability to use and understand
-              English at the university level. And it evaluates how well you
-              combine your reading, listening, speaking and writing skills to
-              perform academic tasks.
-            </p>
-          </li>
-          <li className="process__step process__step_deploy">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deploy">
-            <h3>
-              <spana>3</spana>
-              &mdash;Second place
-            </h3>
-            <p>
-              Got second prize for the competitive algorithm contest of the
-              university
-            </p>
-          </li>
-          <li className="process__step process__step_deliver">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deliver">
-            <h3>
-              <spana>4</spana>
-              &mdash;First place
-            </h3>
-            <p>
-              Highschool, got First prize for the competitive algorithm contest
-              of the city.
-            </p>
-          </li>
-        </ul>
+import React, { useState, Fragment } from 'react'
+import styled from 'styled-components/macro'
+import Icon from '~/views/components/common/Icon'
+import { CERTIFICATIONS } from '~/consts/pages'
+import { INIT_CERT_ACTIVE_ITEM } from '~/consts/initState'
+import PropTypes from 'prop-types'
+
+const Certification = () => {
+  const [itemSelected, setSelectedItem] = useState(INIT_CERT_ACTIVE_ITEM[0])
+  return (
+    <Container id={CERTIFICATIONS}>
+      <div className="container">
+        <header className="common-section__header">
+          <h2>Certifications and Prizes</h2>
+          <h4></h4>
+        </header>
+        <div className="grid-container">
```

Comment:
try to remove this to see if any thing happens, on mobile and desktop

Reply:
Tried to remove <div className="grid-container"> 
no difference between before and after removing

![1](https://user-images.githubusercontent.com/7389488/70730970-81b94c00-1d38-11ea-980e-c54fbd2be1c7.PNG)


![2](https://user-images.githubusercontent.com/7389488/70730983-87af2d00-1d38-11ea-92ea-1cacdfa67835.PNG)

Reply:
ok